lib/src/simple_op_store.rs

```diff
@@ -100,7 +101,7 @@ impl OpStore for SimpleOpStore {
 
         temp_file.as_file().write_all(&proto_bytes)?;
```

Comment:
Since we no longer need `proto_bytes` for hashing, we can clean up a little by writing directly to the file with `proto.write_to_writer(temp_file.as_file())` (that's what I did in #700).